### ISSUE TITLE
Can't create new tween in OnKill callback

### ISSUE THREAD

**Issue by z2yxxx** [Demigiant/dotween#345]
in OnKill Callback call KillAll then create a new tween
the new tween not run, and not trigger any callback

**Comment by Demigiant** [comment]
Hi, on what version are you? I just [released a new update yesterday](http://dotween.demigiant.com/download.php) which might fix this issue.

**Comment by z2yxxx** [comment]
v.1.2.235

**Comment by Demigiant** [comment]
Can you try the latest version and see if the problem goes away?

**Comment by z2yxxx** [comment]
I just upgrade to the latest version and the problem still exist

**Comment by Demigiant** [comment]
Update: I managed to reproduce this and am working on it since your last message but it's... complicated

**Comment by z2yxxx** [comment]
I make a workaround: not use  KillAll, just kill tweens from PlayingTweens list, it works for me

**Comment by Demigiant** [comment]
Yes, the problem is indeed with KillAll called inside an OnKill and then the immediate creation of a tween inside the same OnKill callback, but I'd like that to work :B
